web/src/main/java/org/openmrs/web/WebUtil.java

```diff
@@ -417,23 +416,20 @@ public boolean supportsPropertyName(String propertyName) {
 		return OpenmrsConstants.GP_SEARCH_DATE_DISPLAY_FORMAT.equals(propertyName);
 	}
 	
-	public static void setDefaultDateCache(String defaultDateCache) {
```

Comment:
@teleivo , @dkayiwa 
Thank you for your comments. sorry for delay. Could I know what exact changes I need to do ?